jekyll/_cci2/language-scala.md

```diff
@@ -30,7 +30,7 @@ mkdir .circleci/
 touch .circleci/config.yml
 ```
 
-These commands create a directory named `.circleci` & the next command creates a new file named `config.yml` within the `.circleci` directory.  Again you **must** use the names .circleci for the dir and config.yml.  Learn more about the [version 2.0 prerequisites here](({{ site.baseurl }}/2.0/migrating-from-1-2/#steps-to-configure-required-20-keys).
+These commands create a directory named `.circleci` & the next command creates a new file named `config.yml` within the `.circleci` directory.  Again you **must** use the names .circleci for the dir and config.yml.  Learn more about the [version 2.0 prerequisites here]({{ site.baseurl }}/2.0/migrating-from-1-2/#steps-to-configure-required-20-keys).
```

Comment:
This URL is incorrect, so it doesn't pass the HTML checker test, if you can update the link to /docs/2.0/migrating-from-1-2/ instead, it should build and I will get it merged. Thank you!